tools/azure-plugin/generator/TestProjects/Local/Unbranded-TypeSpec/src/UnbrandedTypeSpec.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This is the UnbrandedTypeSpec client library for developing .NET applications with rich experience.</Description>
+    <AssemblyTitle>SDK Code Generation UnbrandedTypeSpec</AssemblyTitle>
+    <Version>1.0.0-beta.1</Version>
+    <PackageTags>UnbrandedTypeSpec</PackageTags>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.ClientModel" Version="1.1.0-beta.4" />
```

Comment:
This should use the repository global versions.

Reply:
these are generated csprojs so they should not use the global default as the represent what the generator will create which must be able to build stand alone.

Reply:
Please see:  [this comment](https://github.com/Azure/azure-sdk-for-net/pull/45617#discussion_r1729296230)

Reply:
its used to create tests against as well as validating that the project builds after generation in ci.  The tests that use reflection to assert the shape need code to compile against.  This will be the only full library checked in, but there will be stub libraries for all the cadl ranch test as well.  Similar to https://github.com/microsoft/typespec/tree/main/packages/http-client-csharp/generator/TestProjects/CadlRanch